plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts

```diff
@@ -331,7 +320,6 @@ export class SessionManager {
 
             const content = JSON.stringify(messageData) + '\n'
             this.buffer.count += 1
-            this.buffer.size += Buffer.byteLength(content)
```

Comment:
We literally stringify the content above... Couldn't we just opt for using `content.length` here instead? It would be less accurate but not so bad and achieves the actual goal. Swapping to number of lines is adding too much randomness imo